### PR TITLE
Added my (Smit-create) blog

### PR DESCRIPTION
**Pull request by Smit-create**
Added my GSOC blog

### CODE REVIEW

**Comment by Smit-create** [comment]
@czgdp1807 @asmeurer Does this look fine?

**Comment by czgdp1807** [comment]
LGTM

File: planet/planetsympy/config [hunk 1]
```diff
@@ -539,9 +539,15 @@ feed 15m https://www.kangzhiq.com/tag/sympy/feed/
...
         define_faceheight 80
-        
+
 feed 15m https://sc0rpi0n101.github.io/categories/gsoc/index.xml
         define_name Nikhil Maan (Sc0rpi0n101)
         define_face hackergotchi/Nikhil_Maan.jpg
         define_facewidth 80
         define_faceheight 80
+
+feed 15m https://smit-create.github.io/feed.xml
```
Comment (czgdp1807) [inline planet/planetsympy/config:549]:
```suggestion
feed 15m https://smit-create.github.io/rss.xml
```

**Comment by Smit-create** [comment]
@czgdp1807 The build is failing. Should I change it to `feed.xml`?

**Comment by certik** [comment]
Can you merge with the latest master to resolve conflicts?

This will also trigger a new build. The build failure seems unrelated.

@asmeurer there used to be a way to restart a build at Travis-CI, but I couldn't figure out how to do so.

**Comment by certik** [comment]
Thanks!

**Comment by asmeurer** [comment]
I think the UI is the same as it's always been. There is a "restart build" button at the top. Make sure you are logged in, as otherwise it won't let you do it (only people with push access can restart the build).

**Comment by certik** [comment]
I see it now:

Screenshot_2020-05-07 Build #240 - sympy planet-sympy - Travis CI

I would swear it wasn't there before even after I logged in, but maybe I remember it wrong.

Thanks!

**Pull request merged**